### PR TITLE
Corrections and additions to README.md

### PR DESCRIPTION
**Pull request by rowild**
Fix grammar issues.
Add additional git steps.
Add a note about console errors to be ignored.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,11 +113,16 @@ pip install mkdocs mkdocs-material
...
+changes to any file, the server will automatically reload and display your edited version.
 
 ### Build the docs
 
 To build the `aimoes-docs` locally, use `mkdocs build`. You can now open the documentation
 in your browser from the newly generated `<your-aimeos-docs-clone-folder>/docs`´s index.html.
+
+### Troubleshooting
+
+Due to some configuration limits of mkdocs, the command line will print out a 404
```
Comment (aimeos) [inline README.md:126]: That's not entirely correct. Only `mkdocs serve` will tell you that `version.js` isn't found

**Comment by aimeos** [comment]
Can use use *aimeos-docs* (asterisk "*" in front and end) instead of `aimeos-docs`? Usually, `...` is only for code

**Comment by rowild** [comment]
Danke für den Hinweis mit dem Asterisk! Gibt es noch mehrere solche falsch formatierten Begriffe?

**Comment by rowild** [comment]
I guess that means "no"... ;-)

**Pull request merged**